toolkit/scss/shared_placeholders/_dm-typography.scss

```diff
@@ -1,10 +1,30 @@
 @import "_typography.scss";
+@import "_measurements.scss";
 
-%heading-xmedium {
+// All heading styles, except heading-xmedium, are copied from GOVUK Elements as of this commit:
```

Comment:
This could be better please. What do we mean by 'copied'. Is it just in terms of copying class names etc.

heading-small is mentioned here and heading-xmedium. What about things like heading-large? Where is that defined? Is that defined? Why are these different?